xml/cl.xml

```diff
@@ -1087,9 +1087,8 @@ server's OpenCL/api-docs repository.
             document under the section "Registering a Vendor ID with Khronos".
             Only once the ID has been reserved should a new enum entry be added here.
         </comment>
-        <unused start="0x10000" end="0x10003" comment="These Khronos vendor IDs are already assigned in vk.xml"/>
-        <enum value="0x10004" name="CL_KHRONOS_VENDOR_ID_CODEPLAY" comment="Codeplay Software Ltd."/>
```

Comment:
We're using this vendor ID, looks like it's getting removed. Should PoCL be getting added rather than removing Codeplay?

Reply:
Looks good now, thanks